maxim/logger/smolagents/client.py

```
@@ -0,0 +1,539 @@
+            provider = "huggingface"
+        elif 'transformers' in model_class_name:
+            provider = "transformers"
+        elif 'azure' in model_class_name:
+            provider = "azure_openai"
+        elif 'bedrock' in model_class_name:
+            provider = "amazon_bedrock"
+    
+    return model_name, provider
+
+
+def to_serializable_dict(obj):
```

Ruff (0.12.2)

Missing return type annotation for public function `to_serializable_dict`

(ANN201)